CMakeLists.txt

```diff
@@ -664,9 +671,22 @@ INSTALL(TARGETS raw_r
         ARCHIVE DESTINATION lib${LIB_SUFFIX}
        )
 
-# Install find cmake script to the system for client applications.
-INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/FindLibRaw.cmake
-        DESTINATION ${INSTALL_CMAKE_MODULE_PATH})
+IF(MSVC)
+    INSTALL(FILES ${PROJECT_BINARY_DIR}/raw.pdb
+            DESTINATION lib
+            CONFIGURATIONS Debug RelWithDebInfo
+           )
+    INSTALL(FILES ${PROJECT_BINARY_DIR}/raw_r.pdb
+            DESTINATION lib
+            CONFIGURATIONS Debug RelWithDebInfo
+           )
+ENDIF()
+
+IF(INSTALL_CMAKE_SCRIPT)
+    # Install find cmake script to the system for client applications.
+    INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/FindLibRaw.cmake
+            DESTINATION ${INSTALL_CMAKE_MODULE_PATH})
```

Comment:
What is `INSTALL_CMAKE_MODULE_PATH`? Seems undefined.

Reply:
Thats from the existing code.
I've just moved that section of code into an "IF(INSTALL_CMAKE_SCRIPT)" block.

Reply:
I see. Let's fix that though because it doesn't make much sense. I would remove `INSTALL_CMAKE_SCRIPT` again and rather make it `DESTINATION share/libraw/cmake` since that's one of the locations CMake will search for the script: https://cmake.org/cmake/help/v3.6/command/find_package.html

Reply:
I just checked the file again, and INSTALL_CMAKE_MODULE_PATH is already defined earlier in the file, line 111:

```
SET(INSTALL_CMAKE_MODULE_PATH  "${CMAKE_ROOT}/Modules"  CACHE STRING
    "Path to install cmake module              (default=${CMAKE_ROOT}/Modules)")
```

Reply:
OK, this is still incorrect and should be changed. `${CMAKE_ROOT}/Modules` are the built-in modules files which shouldn't be touched. I suggest changing the default to `share/libraw/cmake` and removing `INSTALL_CMAKE_SCRIPT`, except if there's a good reason to keep it.